test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,49 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+     When executing "minishift profile set remote" succeeds
+      And executing "minishift start --no-provision" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Provisioning remote instance
+    Given unset environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" succeeds
+      And setting scenario variable "REMOTE_VM_IP" to the stdout from executing "minishift ip"
```

Comment:
This is nice :+1:

Reply:
@gbraad I use it sometimes for debugging also, I appreciate this particular step a lot :)

Reply:
good to hear, as there was pushback at the time of implementation. As it does not really honor environment variables. But this is actually the point, as it should be helpful to verify external variables against those we gather in the scenario, and vice versa. but also, to re-use information such as IP address, names, etc

Perhaps there is more we can do with this?